config/features.yml

```diff
@@ -416,6 +416,10 @@ features:
     actor_type: user
     description: Finds facilities using VAOS's Mobile Facility Service.
     enable_in_development: false
+  mobile_precache_appointments:
```

Comment:
How does this work for staging? Will we be able to test the flipper in staging?

Reply:
I believe it will default to off and I will have to turn the flag on after this code is deployed.